client/src/helpers/stardust/preExpandedConfig.ts

```diff
@@ -45,7 +45,8 @@ export function getInputsPreExpandedConfig(inputs: IInput[], unlocks: Unlock[],
             if (input?.output?.output && "unlockConditions" in input.output.output) {
                 const commmonOutput = input.output.output as unknown as CommonOutput;
                 let unlock = unlocks[idx];
-                if (unlock.type === UnlockType.Reference) {
+                const referenceUnlocks = [UnlockType.Reference, UnlockType.Alias, UnlockType.Nft];
```

Comment:
There are more `UnlockType`s that this though ? Shouldn't we cover all of them.
@begonaalvarezd Apart from that, now that i red it better, not sure that this logic is solid, cause "referenceUnlocks" can be transitive.
Check how we do it in `client/src/helpers/stardust/transactionsHelper.ts` (`getInputsAndOutputs`).

Reply:
![image](https://github.com/iotaledger/explorer/assets/3624944/8a54c7f6-7fb6-4227-8804-b0ce759ee126)